platform.js

```diff
@@ -415,12 +416,12 @@ module.exports = {
         });
       }
       else {
-        var lnxCommand = "df --block-size=K --output=avail " + module.exports.getInstallDir();
+        var lnxCommand = "df -k " + dir + " | awk 'NR==2 {print $4}'";
 
```

Comment:
@rodrigopavezi Did you test these changes on Ubuntu 14.04 and confirm they still work?